src/Auth0.ManagementApi/Models/ClientBase.cs

```diff
@@ -119,9 +119,11 @@ public abstract class ClientBase
         [JsonProperty("name")]
         public string Name { get; set; }
 
-
+        /// <summary>
+        /// 
```

Comment:
How about `A list of resource servers (APIs) that the client is authorized to request access tokens for, using the Client Credentials exchange.`?